test/util/chat.go

```diff
@@ -74,6 +75,41 @@ func CreateChat(client HTTPClient, baseURL string, token string, createChatReque
 	return &createChatResponse
 }
 
+func UpdateChat(client HTTPClient, baseURL string, token string, updateChatRequest *chathttp.UpdateChatDto, status int) *chathttp.ChatDto {
+	requestBody, err := json.Marshal(updateChatRequest)
+	gomega.ExpectWithOffset(1, err).NotTo(gomega.HaveOccurred())
+
+	req, err := http.NewRequest(http.MethodPut, fmt.Sprintf("%s/chats/%d", baseURL, updateChatRequest.ID), bytes.NewBuffer(requestBody))
+	gomega.ExpectWithOffset(1, err).ToNot(gomega.HaveOccurred())
+
+	req.Header.Set("Authorization", fmt.Sprintf("Bearer %s", token))
+	req.Header.Set("Content-Type", "application/json")
+
+	resp, err := client.Do(req)
+	gomega.ExpectWithOffset(1, err).ToNot(gomega.HaveOccurred())
+	defer resp.Body.Close()
+
+	gomega.ExpectWithOffset(1, resp.StatusCode).To(gomega.Equal(status))
+
+	responseBody, err := io.ReadAll(resp.Body)
+	gomega.ExpectWithOffset(1, err).ToNot(gomega.HaveOccurred())
+
+	gomega.ExpectWithOffset(2, resp.StatusCode).To(gomega.Equal(status))
+
+	var updateChatResponse chathttp.ChatDto
+	gomega.ExpectWithOffset(1, json.Unmarshal(responseBody, &updateChatResponse)).To(gomega.Succeed())
+
+	return &updateChatResponse
+}
+
+func RemoveAllChats(client HTTPClient, baseURL string, token string) {
+	chats := GetChats(client, baseURL, token)
+
+	for _, chat := range chats.Items {
+		DeleteChat(client, baseURL, token, chat.ID)
```

Comment:
```suggestion
		deleteChat(client, baseURL, token, chat.ID, http.StatusOK)
```
Use internal `deleteChat` to have correct offset.